### ISSUE TITLE
Is app: now a historical scheme?

### ISSUE THREAD

**Issue by stain** [sysapps/app-uri#44]
I see from #24 that `app` didn't make it into [IANA's URI schemes](https://www.iana.org/assignments/uri-schemes/), even as a [provisional registration](https://tools.ietf.org/html/bcp35#section-4).

http://www.w3.org/TR/2015/NOTE-app-uri-20150723/ now sadly says:

> This specification is being republished as a Working Group Note as an indication that it not being progressed further as a Recommendation track document. 

I guess this follows from [closing of the sysapps WG](https://lists.w3.org/Archives/Public/public-sysapps/2015Mar/0009.html)

Does this mean that `app` URI scheme is now defunct, or is it something others might be thinking about moving forwards?

We had - perhaps prematurely - [used app:](https://w3id.org/bundle/#absolute-uris) as part of Research Object Bundle identifiers - generating a unique identifier for a resource within a structured ZIP file.

Are there other de-facto users or historical users of `app://` URIs you know about?

Would it make sense now to register `app` with IANA as a [historical](https://tools.ietf.org/html/bcp35#section-5) scheme? I can be the individual.

**Comment by marcoscaceres** [comment]
Yeah, it’s defunct. I don’t think we should bother registering it as historical. Someone else might want it and they should be free to repurpose it.

**Comment by stain** [comment]
Perhaps I can try to repurpose only the identifier bit as a provisional registration;  not all the pretend-HTTP details which may have scared IETF folks.

I think it was a great idea, not just for navigation within mobile apps; we used it for instance when parsing multiple RDF files (with relative hyperlinks) from inside a ZIP file or from other non-web sources like SQL databases, but also to generate global identifiers based on hashing the archive file (both of these are tricky with other schemes).

**Comment by stain** [comment]
I started an Internet-Draft at https://github.com/stain/I-D/blob/master/app/app.md aka https://rawgit.com/stain/I-D/master/app/app.html -- any brief comments, @marcoscaceres ?

**Comment by stain** [comment]
Submitted to [uri-review](https://mailarchive.ietf.org/arch/msg/uri-review/B2OYkX7_BTs4EBtH0A8Y_4hPkdQ)
and https://tools.ietf.org/html/draft-soilandreyes-app-00

Let me know if you would like to be co-author? That would only be fair.

**Comment by marcoscaceres** [comment]
@stain, thanks for putting that together! Happy to be listed as co-author.

**Comment by stain** [comment]
Cool! Should soon appear on https://tools.ietf.org/html/draft-soilandreyes-app-01 -- feel free to pull-request/issue on https://github.com/stain/I-D/blob/master/app/app.md

Note the retronym for `app`: "The Archive and Packaging Pointer system"
